src/adjtrans.jl

```diff
@@ -382,6 +382,20 @@ parent(A::AdjOrTrans) = A.parent
 vec(v::TransposeAbsVec{<:Number}) = parent(v)
 vec(v::AdjointAbsVec{<:Real}) = parent(v)
 
+Base.reshape(v::TransposeAbsVec{<:Number}, ::Val{1}) = parent(v)
+Base.reshape(v::AdjointAbsVec{<:Real}, ::Val{1}) = parent(v)
+
+ # these make eachrow(A') produce simpler views
+@inline Base.unsafe_view(A::Transpose{<:Number, <:AbstractMatrix}, i::Integer, j::AbstractArray) =
```

Comment:
Perhaps use the alias `TransposeAbsMat` like the `TransposeAbsVec` alias used in `reshape`?

Reply:
IDK why it's written this way, it's possible I followed nearby code which writes e.g. `Base.strides(A::Adjoint{<:Real, <:AbstractMatrix})` and does not seem to use TransposeAbsMat. Maybe I was concerned about order of definitions or something?